test/utils/helperFn.ts

```diff
@@ -49,3 +49,23 @@ export function getColumnHeaders() {
     (node) => node!.textContent,
   );
 }
+export function getCell(rowIndex: number, colIndex: number): HTMLElement {
+  const cell = document.querySelector(
+    `[role="cell"][data-rowindex="${rowIndex}"][aria-colindex="${colIndex}"]`,
+  );
+  if (cell == null) {
+    throw new Error(`Cell ${rowIndex} ${colIndex} not found`);
+  }
+  return cell as HTMLElement;
+}
+export function focusCell(rowIndex: number, colIndex: number) {
```

Comment:
I think that it's important we keep the usage of the test helpers to their bare minimum. The fewer we can get away with, the more duplication we can have between tests, the more reliable and helpful they will be.

In this case, the focusCell looks redundant, there are no complexity worth abstracting.

```suggestion
```

Reply:
it avoids doing `getCell(0, 0).focus()` 
It looks nice and easy to read.
Also I was thinking of moving the check active element  or even deleting it